util/verible-format.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# this script runs the verible formatter on all system verilog
+# files under hw/{ip,vendor,top_earlgrey}
+#
+# make sure to invoke this tool from the project root.
+#
+# NOTE: this operates in-place - so make sure to make a backup or
+# run this on an experimental branch
+#
+# TODO: integrate this with Fusesoc and the other linting flows.
+
+NUM_PROCS=8
+REPORT_FILE="verible-style-format.rpt"
+
+# get all system verilog files and pipe through style formatter
```

Comment:
Just a word of caution for any new formatter tool, might recommend `git add` on all files in the workspace before trying the formatter, so that in case it botches up files, you can quickly rollback to a known-good state of files.

Reply:
Good point. Let me add that.